examples/run/run.cpp

```diff
@@ -1053,11 +1074,43 @@ static int get_user_input(std::string & user_input, const std::string & user) {
     return 0;
 }
 
+// Reads a chat template file to be used
+static std::string read_chat_template_file(const std::string & chat_template_file) {
+    if(chat_template_file.empty()){
+        return "";
+    }
+
+    FILE* file = ggml_fopen(chat_template_file.c_str(), "r");
+        if (!file) {
+        std::cerr << "Error opening chat template file '" << chat_template_file << "': " << strerror(errno) << "\n";
```

Comment:
This would be preferred:

```
printe("Error opening chat template file '%s': %s\n", chat_template_file.c_str(), strerror(errno));
```

we don't use cout, cerr here.

Reply:
I'd also clang-format this, indenting is off:

```
        if (!file) {
        std::cerr << "Error opening chat template file '" << chat_template_file << "': " << strerror(errno) << "\n";
```

Reply:
Good point! Changed.